packages/core/src/components/text/Text.tsx

```diff
@@ -12,6 +12,7 @@ export interface TextProps extends SpanProps {
   whiteSpace?: Property.WhiteSpace;
   wordBreak?: Property.WordBreak;
   color?: string;
+  fontWeight?: Property.FontWeight;
```

Comment:
`fontWeight` should not be part of `Text`. 
The different looks of text comes from `variant`.